cmd/svcat/output/yaml.go

```diff
@@ -59,3 +60,16 @@ func writeParameters(w io.Writer, parameters *runtime.RawExtension) {
 		writeYAML(w, params, 2)
 	}
 }
+
+func writeParametersFrom(w io.Writer, parametersFrom []v1beta1.ParametersFromSource) {
+	if parametersFrom == nil || len(parametersFrom) == 0 {
+		return
+	}
+
+	fmt.Fprintln(w, "\nParameters From:")
+	for _, p := range parametersFrom {
+		fmt.Fprintln(w, "  Secret Key Ref:")
+		fmt.Fprintf(w, "    Name: %s\n", p.SecretKeyRef.Name)
```

Comment:
Somewhat opposite to above, SecretKeyRef could be nil here so check for that before attempting to print. (This also means that potentially there may be no output, yet the "Parameters From:" header would still get printed, which would look weird.)

Reply:
Thanks. I missed the fact that `SecretKeyRef`is optional.